apps/meteor/server/models/raw/LivechatContacts.ts

```diff
@@ -9,6 +10,30 @@ export class LivechatContactsRaw extends BaseRaw<ILivechatContact> implements IL
 		super(db, 'livechat_contact', trash);
 	}
 
+	protected modelIndexes(): IndexDescription[] {
+		return [
+			{
+				key: { name: 1 },
+				unique: false,
+				sparse: true,
+				name: 'name_insensitive',
+				collation: { locale: 'en', strength: 2, caseLevel: false },
+			},
+			{
+				key: { emails: 1 },
+				unique: false,
+				sparse: true,
+				name: 'emails_insensitive',
+				collation: { locale: 'en', strength: 2, caseLevel: false },
+			},
+			{
+				key: { phones: 1 },
+				sparse: true,
```

Comment:
https://www.mongodb.com/docs/manual/core/index-partial/#comparison-with-sparse-indexes

Prefer partial indexes over sparse indexes. If we can use a partial here, it would be better